xml/ha_clvm.xml

```diff
@@ -348,6 +345,139 @@ cLVM) for more information and details to integrate here - really helpful-->
   /dev/sdb vgtest b</screen>
     </step>
    </procedure>
+   <para>To test your clustered mirror device, proceed as follows:</para>
```

Comment:
@renzhengeek Thanks for your feedback, much appreciated! 👍

OK, I fixed the typo and added the two more devices (see commit 0b0eddf). For the procedure, I've used your setup from https://github.com/SUSE/doc-sleha/pull/36#discussion_r124184979 (see commit b1f81a6).

What is still a little bit unclear to me is, what should I do with procedure "pro.ha.clvm.config.test"? Should I leave it or remove it?

If procedure `pro.ha.clvm.config.test` is still useful, could you answer the `remark` from line [L442](https://github.com/SUSE/doc-sleha/pull/36/files#diff-8c7fd2de86d5011a147e958055aed85dR430)?

Many thanks! 👍